skylines/lib/emails.py

```diff
@@ -0,0 +1,35 @@
+from threading import Thread
+from flask import current_app, copy_current_request_context
+from flask_mail import Message
+
+
+def send_async(msg):
```

Comment:
Have a look at https://github.com/skylines-project/skylines/blob/master/skylines/worker/tasks.py#L12 and https://github.com/skylines-project/skylines/blob/master/skylines/frontend/views/upload.py#L418 how to invoke it. It's quite simple.